src/modules/dashboard/components/ExpenditurePage/Streaming/LockedStreaming.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import { FormSection, InputLabel } from '~core/Fields';
+import UserAvatar from '~core/UserAvatar';
+import UserMention from '~core/UserMention';
+import { useLoggedInUser } from '~data/index';
+
+import styles from './LockedStreaming.css';
+
+export const MSG = defineMessages({
+  type: {
+    id: 'dashboard.ExpenditurePage.Streaming.LockedStreaming.type',
+    defaultMessage: 'Expenditure type',
+  },
+  streaming: {
+    id: 'dashboard.ExpenditurePage.Streaming.LockedStreaming.streaming',
+    defaultMessage: 'Streaming',
+  },
+  owner: {
+    id: 'dashboard.ExpenditurePage.Streaming.LockedStreaming.owner',
+    defaultMessage: 'Owner',
+  },
```

Comment:
It is "To" before the locked state, but changes to "Owner" once it's been locked.

Reply:
yes, that is how it behaves, when the form is editable there is a `To` information, once the form is locked it's `Owner`

Reply:
@joannasia I don't see that in the [design](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=5740%3A333808). Can you point me to the screen where it says "owner"?

Reply:
@willm30 i think i misunderstood it completely, you're right, it should say _To_, it is fixed now, thanks!